app/models/account.rb

```diff
@@ -28,13 +29,28 @@ class Account < ApplicationRecord
   include Featurable
   include CacheKeys
 
+  SETTINGS_PARAMS_SCHEMA = {
+    'type': 'object',
+    'properties':
+      {
+        'auto_resolve_after': { 'type': %w[integer null], 'minimum': 10, 'maximum': 1_439_856 },
```

Comment:
I notice that the `auto_resolve_after` value is set to 15 minutes in the description and tests. Should we update this value here to match instead of keeping it at 10?

Reply:
10 seemed fair, I'll update the values elsewhere

Reply:
@muhsin-k where else did you find the `15` reference?